docs/demos/coast-to-coast-q4-2023.md

```diff
@@ -0,0 +1,793 @@
+- Generate the manifests
+```sh
+  make generate manifests
+```
+
+- Build the controller image
+```sh
+  make docker-build IMG="quay.io/operator-framework/coastal:{VERSION}"
+```
+
+- Populate the `manifests/` directory for building the plain bundle
+```sh
+  kustomize build config/default > manifests/manifests.yaml
+```
+
+- Build the plain bundle image
+```sh
+docker build -t quay.io/operator-framework/coastal-bundle:{VERSION} -f plainbundle.Dockerfile .
+```
+
+- Push the controller image and plain bundle image
+```sh
+docker push quay.io/operator-framework/coastal:{VERSION} && \
+docker push quay.io/operator-framework/coastal-bundle:{VERSION}
+```
+
+A handy little script to run all these steps for you:
+```sh
+#!/bin/bash
+set -e
+
+mkdir -p manifests
+
+cat << EOF > plainbundle.Dockerfile
+FROM scratch
+ADD manifests /manifests
+EOF
+
+versions=( v1.0.0-alpha1 v1.0.0 v1.0.1 v1.1.0 v2.0.0 )
+for version in "${versions[@]}"
+do
+  make generate manifests
+  make docker-build IMG="quay.io/operator-framework/coastal:${version}"
+  mkdir -p manifests
+  kustomize build config/default > manifests/manifests.yaml
+  docker build -t "quay.io/operator-framework/coastal-bundle:${version}" -f plainbundle.Dockerfile .
+  docker push "quay.io/operator-framework/coastal:${version}"
+  docker push "quay.io/operator-framework/coastal-bundle:${version}"
+done
+```
+
+### Create an initial File-Based Catalog (FBC) Image
+Now we need to build and push an initial File-Based Catalog (FBC) image containing
+our bundles. To help highlight the new polling functionality, we are only going to add the `v1.0.0-alpha1`
+bundle to our catalog image in this step and as we go through the rest of the sections we will progressively
+add more of our bundles to the catalog.
+
+- Create a `catalog/` directory
+```sh
+mkdir catalog
+```
+
+- Create a Dockerfile for the FBC image
+```sh
+cat << EOF > catalog.Dockerfile
+FROM scratch
+ADD catalog /configs
+LABEL operators.operatorframework.io.index.configs.v1=/configs
+EOF
+```
+
+- Create the FBC YAML file
+```sh
+cat << EOF > catalog/index.yaml
+{
+  "schema": "olm.package",
+  "name": "coastal"
+}
+{
+  "schema": "olm.bundle",
+  "name": "coastal.v1.0.0-alpha1",
+  "package": "coastal",
+  "image": "quay.io/operator-framework/coastal-bundle:v1.0.0-alpha1",
+  "properties": [
+    {
+      "type": "olm.package",
+      "value": {
+        "packageName": "coastal",
+        "version": "1.0.0-alpha1"
+      }
+    },
+    {
+      "type": "olm.bundle.mediatype",
+      "value": "plain+v0"
+    }
+  ]
+}
+{
+  "schema": "olm.channel",
+  "name": "stable",
+  "package": "coastal",
+  "entries": [
+    {
+      "name": "coastal.v1.0.0-alpha1"
+    }
+  ]
+}
+EOF
+```
+
+- Build and push the FBC image
+```sh
+docker build -t quay.io/operator-framework/coastal-catalog:latest -f catalog.Dockerfile . && \
+docker push quay.io/operator-framework/coastal-catalog:latest
+```
+
+### Create a `Catalog`
```

Comment:
```suggestion
### Create a `Catalog` Resource
```

I think I get where you were going here: having a `catalog` composed of FBC which is represented by a `Catalog` resource type could be really confusing.  The above suggestion is my first stab at disambiguating the two by consistently saying 'resource' when that's the focus.

Reply:
[9cecd82](https://github.com/operator-framework/operator-controller/pull/533/commits/9cecd82c89aa82af486c15b987cc97833517ffbf)